subworkflows/local/annotate_snvs.nf

```diff
@@ -64,10 +59,25 @@ workflow ANNOTATE_SNVS {
 
         RHOCALL_ANNOTATE (vcf, ch_roh_rhocall, [])
 
-        TABIX_SNV_ANNO (RHOCALL_ANNOTATE.out.vcf)
-        ch_versions = ch_versions.mix(TABIX_SNV_ANNO.out.versions)
+        TABIX_ROHCALL (RHOCALL_ANNOTATE.out.vcf)
+        ch_versions = ch_versions.mix(TABIX_ROHCALL.out.versions)
 
-        ENSEMBLVEP_SNV(RHOCALL_ANNOTATE.out.vcf,
+        //
+        // annotate vcfanno
+        //
+        VCFANNO (TABIX_ROHCALL.out.gz_tbi, ch_toml, vcfanno_resource_dir)
+        ch_versions = ch_versions.mix(VCFANNO.out.versions)
+
+        TABIX_VCFANNO (VCFANNO.out.vcf)
+        ch_versions = ch_versions.mix(TABIX_VCFANNO.out.versions)
+
+        BCFTOOLS_VIEW(TABIX_VCFANNO.out.gz_tbi,[],[],[])
+        ch_versions = ch_versions.mix(BCFTOOLS_VIEW.out.versions)
```

Comment:
Is this necessary? VEP can take a bgzipped output

Reply:
This is me compressing the ungzipped file from vcfanno. This file is then fed into vep.